modules/customer-magento/magento/afterResponse/useCheckCacheId.ts

```diff
@@ -0,0 +1,17 @@
+import useCustomerMagentoStore from '#ioc/stores/useCustomerMagentoStore'
+
+export default () => {
```

Comment:
Move to magento module and rename to useGetCacheId or useStoreCacheId or something like that.